pom.xml

```diff
@@ -95,8 +95,8 @@
     </mailingLists>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>16</maven.compiler.source>
+        <maven.compiler.target>16</maven.compiler.target>
```

Comment:
Is there a reason these are 16 instead of 17?

Reply:
No, it can be 17, updated.